tests/integration/dynamic_ip_test.py

```diff
@@ -839,12 +845,17 @@ def dummy00():
     libnmstate.apply({Interface.KEY: [ifstate]}, verify_change=False)
 
 
-def test_activate_dummy_without_dhcp_service(dummy00):
+@parametrize_ip_ver
+def test_activate_dummy_without_dhcp_service(ip_ver, dummy00):
+    if ip_ver != (Interface.IPV4, Interface.IPV6):
+        pytest.xfail('https://bugzilla.redhat.com/1798374', strict=True)
```

Comment:
You should place the xfails in the `parametrize`.

Reply:
Done, thanks